packages/app-cli/app/command-mkbook.test.ts

```diff
@@ -0,0 +1,32 @@
+import { setupDatabaseAndSynchronizer, switchClient } from '@joplin/lib/testing/test-utils';
+import { setupCommandForTesting, setupApplication } from './utils/testUtils';
+const Command = require('./command-mkbook');
+
+
+describe('command-mkbook', function() {
+
+	beforeEach(async () => {
+		await setupDatabaseAndSynchronizer(1);
+		await switchClient(1);
+		await setupApplication();
+	});
+
+	const command = setupCommandForTesting(Command);
```

Comment:
There shouldn't be setup code at this level because it means the tests are no longer independent from each others  - it should either be in beforeEach or within each test.